CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 The GraphQL Accounts project was intended - since it's inception - to be a community maintained project.  We'd love to see you get involved (especially long time contributors from the Meteor community who we've worked with before).
```

Comment:
I'd be happy to. I was in the process of adding getting started stuff [here](https://github.com/KyleScharnhorst/accounts/blob/add_getting_started_steps/CONTRIBUTING.md)